### ISSUE TITLE
1.20.1 Fabric - Sodium causes rendering issues

### ISSUE THREAD

**Issue by Watsof** [jaquadro/StorageDrawers#1168]
Installing Storage Drawers alongside Sodium (base mod only, no addons) causes the following visual issues:

- The keyhole and eye of ender icons don't show up when a key is used. Respective key's functionality is not affected, both drawer locking and concealment work as intended.

- The front texture doesn't change when a drawer puller is used.

I haven't found a Sodium setting that fixes these conflicts.

EDIT: Same problem with the voiding upgrade icon. I'd imagine it affects all icons rendered in the same way.

Tested on:
Fabric 1.20.1
Storage drawers 12.10.1
Sodium 0.5.11

**Comment by jaquadro** [comment]
I'm still not that familiar with fabric ecosystems, but I recall reading messages about needing another mod asking with sodium when the fabric rendering API was in use, maybe called indium?

**Comment by Watsof** [comment]
Uh, you're right. Indium does solve the issue.
I didn't suspect it because it's installed on my main Fabric profile too, but now I discovered some leftover config file from other/older mods was conflicting and disabling parts of Indium? Regenerating the configs solved it, along with a bunch of stuff I didn't even realize was off in the first place. Weird.
Guess I need to clean up my modded profile.

TL;DR: Indium is required with sodium 0.5 (duh). Sodium 0.6 and above has it already integrated.

**Comment by iamtokimi** [comment]
I'm having this same issue within my modpack and I have Indium installed so I'm not sure what is causing it.
